### PR TITLE
CodeSector.TeraCopy version 3.9.0

### PR DESCRIPTION
**Pull request by brotich**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/42086)

### CODE REVIEW

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CodeSector.TeraCopy
+PackageVersion: 3.9
```
Comment (ItzLevvie) [inline manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.9.0
```

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: CodeSector.TeraCopy
+PackageVersion: 3.9
```
Comment (ItzLevvie) [inline manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml:5]:
```suggestion
PackageVersion: 3.9.0
```

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CodeSector.TeraCopy
+PackageVersion: 3.9
```
Comment (ItzLevvie) [inline manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.yaml:5]:
```suggestion
PackageVersion: 3.9.0
```

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://codesector.s3.us-west-000.backblazeb2.com/teracopy3.9.exe
+  InstallerSha256: 9DFA57C6DD935F72E7E8254146CD143528FE962D2C2F518C2D6511D4D1150A3A
+  InstallerSwitches:
+    Silent: /q
+    SilentWithProgress: /qb
+  UpgradeBehavior: install
+  ProductCode: '{952ED35A-74C3-4204-8F01-986D8711B41D}'
```
Comment (ItzLevvie) [inline manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml:22]:
```suggestion
  ProductCode: '{F8B0BB18-B1E6-4821-8C5B-883AA5DE3EEA}'
```

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml:5]: This must match Control Panel.

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.yaml:5]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will need to update the version folder from `3.9` to `3.9.0` using Visual Studio Code/Online by pressing the `.` (dot) on your keyboard on this PR.

File: manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml (hunk 4 above)
Comment (brotich) [reply manifests/c/CodeSector/TeraCopy/3.9/CodeSector.TeraCopy.installer.yaml]: i do not understand whey the product code is coming from, and why it is different from the earlier version here. where did you get this product code `F8B0BB18-B1E6-4821-8C5B-883AA5DE3EEA`?

File: manifests/c/CodeSector/TeraCopy/3.9.0/CodeSector.TeraCopy.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://codesector.s3.us-west-000.backblazeb2.com/teracopy3.9.exe
+  InstallerSha256: 9DFA57C6DD935F72E7E8254146CD143528FE962D2C2F518C2D6511D4D1150A3A
+  InstallerSwitches:
+    Silent: /q
+    SilentWithProgress: /qb
+  UpgradeBehavior: install
+  ProductCode: '{952ED35A-74C3-4204-8F01-986D8711B41D}'
```
Comment (ItzLevvie) [inline manifests/c/CodeSector/TeraCopy/3.9.0/CodeSector.TeraCopy.installer.yaml:22]:
```suggestion
  ProductCode: '{F8B0BB18-B1E6-4821-8C5B-883AA5DE3EEA}'
```

> i do not understand whey the product code is coming from, and why it is different from the earlier version here. where did you get this product code `F8B0BB18-B1E6-4821-8C5B-883AA5DE3EEA`?

It's taken from the Windows Registry.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/c/CodeSector/TeraCopy/3.9.0/CodeSector.TeraCopy.installer.yaml (hunk 5 above)
Comment (brotich) [reply manifests/c/CodeSector/TeraCopy/3.9.0/CodeSector.TeraCopy.installer.yaml]: okay. i wasnt ware of this. thanks

**Pull request merged**